data/migrations/20201109234734_seller_profie_and_item.js

```diff
@@ -20,9 +20,19 @@ exports.up = function (knex) {
       tb.increments();
       tb.string('category_name', 255);
     })
-    .createTable('tag', (tb) => {
+    .createTable('favorite_category', (tb) => {
       tb.increments();
-      tb.string('tag_name', 255);
+      tb.string('categories', 255)
```

Comment:
sorry, should have noted this in my first comment, but can you switch that to the category id?

Reply:
fixed!